framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -894,7 +892,11 @@ public AnnotationFormatter getAnnotationFormatter() {
      *         empty set if no qualifiers are supported
      */
     public final Set<Class<? extends Annotation>> getSupportedTypeQualifiers() {
```

Comment:
Please add a comment to the declaration of `supportedQuals` stating that getSupportedTypeQualifiers should be used instead of use supportedQuals directly.